parser/src/ast/evaluator_function.rs

```diff
@@ -1,27 +1,25 @@
-use super::{Identifier, IntegrityStmt, TraceCols};
+use super::{Identifier, IntegrityStmt, TraceBinding, TraceBindingAccess};
 
-/// Evaluator functions take descriptions of the main and auxiliary traces as input, and enforce
+/// Evaluator functions take column groups as parameters where each column group is a set of
+/// columns in the particular trace segment that are passed to the evaluator function, and enforce
```

Comment:
nit: we should update the terminology in this comment from column groups to trace bindings